cluster/ci/config/roles.yaml

```diff
@@ -142,6 +157,19 @@ items:
     apiGroup: rbac.authorization.k8s.io
     name: ci-admins
   roleRef:
-    kind: Role
+    kind: ClusterRole
```

Comment:
@kargakis this is related to ci/admin, but I guess there should be ClusterRole too.

Reply:
Yeah, I think this is correct.